manager/notifier.go

```diff
@@ -494,6 +563,15 @@ func (n *notifier) handleNotification(a *Alert, op notificationOp, config *pb.No
 			glog.Errorln("Error sending Slack notification:", err)
 		}
 	}
+	for _, fdConfig := range config.FlowdockConfig {
+		if op == notificationOpResolve && !fdConfig.GetSendResolved() {
+			continue
+		}
+		flowdockMessage := newFlowdockNotificationMessage(op, fdConfig, a)
+		url := *flowdockURL + "/" + fdConfig.GetApiToken()
+		httpResponse := postJSON(marshallJSON(flowdockMessage), url)
```

Comment:
Why would we want to make the request with invalid data?
I don't think the the `marshallJSON` helper is beneficial in general. It just hides the error from the control flow.